Olympus.Stratis/functions/fn_featureOverview.sqf

```diff
@@ -20,6 +20,11 @@ _player createDiaryRecord ["features", ["Environment Controller", "
     <br/>The main base and some courses have an additional laptop to control the environment.
 "]];
 
+_player createDiaryRecord ["features", ["Vehicle Spawner", "
+    <br/><img image='x\tacr\addons\images\olympus\feature\vehicleSpawn.jpg' width='300' height='350'/>
```

Comment:
Currently has an issue with this image, it won't even work on a laptop (while the teleporter one from #127 works fine.

Reply:
Solved, it was a corrupted image (please never use paint again @TyroneMF)